.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -37,291 +34,415 @@ jobs:
-        set -x
-        # With GCC 13, LLVM fails some tests, then we use GCC 12 instead
-        export GCC_VERSION=12
-        docker run --rm --platform linux/aarch64 \
-            -v /var/run/docker.sock:/var/run/docker.sock \
-            -v `pwd`:/catalyst \
-            -i ${{ matrix.container_img }} \
-            bash /catalyst/.github/workflows/scripts/linux_arm64/rh8/build_llvm.sh $GCC_VERSION ${{ matrix.python_version.major_minor }} ${{ matrix.python_version.patch }} ${{ matrix.python_version.package }}
-
-    - name: Save LLVM Build
-      id: save-llvm-build
-      if: steps.cache-llvm-build.outputs.cache-hit != 'true'
-      uses: actions/cache/save@v4
-      with:
-        path: ${{ github.workspace }}/llvm-build
-        key: ${{ matrix.container_name }}-llvm-${{ needs.constants.outputs.llvm_version }}-wheel-build
-
-    - name: Build MHLO Dialect
-      if: steps.cache-mhlo-build.outputs.cache-hit != 'true'
-      run: |
-        set -x
-        export GCC_VERSION=13
-        docker run --rm --platform linux/aarch64 \
-            -v /var/run/docker.sock:/var/run/docker.sock \
-            -v `pwd`:/catalyst \
-            -i ${{ matrix.container_img }} \
-            bash /catalyst/.github/workflows/scripts/linux_arm64/rh8/build_mhlo.sh $GCC_VERSION ${{ matrix.python_version.major_minor }} ${{ matrix.python_version.patch }} ${{ matrix.python_version.package }}
-
-    - name: Save MHLO Build
-      id: save-mhlo-build
-      if: steps.cache-mhlo-build.outputs.cache-hit != 'true'
-      uses: actions/cache/save@v4
-      with:
-        path: ${{ github.workspace }}/mhlo-build
-        key: ${{ matrix.container_name }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-wheel-build
-
-    - name: Build Enzyme
-      if: steps.cache-enzyme-build.outputs.cache-hit != 'true'
-      run: |
-        set -x
-        export GCC_VERSION=13
-        docker run --rm --platform linux/aarch64 \
-            -v /var/run/docker.sock:/var/run/docker.sock \
-            -v `pwd`:/catalyst \
-            -i ${{ matrix.container_img }} \
-            bash /catalyst/.github/workflows/scripts/linux_arm64/rh8/build_enzyme.sh $GCC_VERSION ${{ matrix.python_version.major_minor }} ${{ matrix.python_version.patch }} ${{ matrix.python_version.package }}
-
-    - name: Save Enzyme Build
-      id: save-enzyme-build
-      if: steps.cache-enzyme-build.outputs.cache-hit != 'true'
-      uses: actions/cache/save@v4
-      with:
-        path: ${{ github.workspace }}/enzyme-build
-        key: ${{ matrix.container_name }}-enzyme-${{ needs.constants.outputs.llvm_version }}-${{ needs.constants.outputs.enzyme_version }}-wheel-build
-
-  catalyst-linux-wheels-arm64:
+    steps:  
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Cache LLVM Source
+        id: cache-llvm-source
+        uses: actions/cache@v4
+        with:
+          path: ${{ github.workspace }}/mlir/llvm-project
```

Comment:
```suggestion
          path: mlir/llvm-project
```

We had `${{ github.workspace }}` in the path cause it could be different, as we had multiple runners on the same machine.

Now with github runners, we should be okay to remove this from the path

Reply:
I think it's fine, we use that prefix in all the paths in our wheel build actions so we might as well leave it to reduce the amount of diff introduced.